src/format_otr/export_txt.py

```diff
@@ -0,0 +1,48 @@
+from bs4 import BeautifulSoup
+from pathlib import Path
+import json
+
+
+def get_text_between(start, end):
+    text = ''
+    tag = start.next_element.next_element
+    while tag != end:
+        if tag.name is None:
+            text += tag.text
+        tag = tag.next_element
+    return text
+
+def get_text_till_end(start):
+    text = ''
+    tag = start.next_element.next_element
+    while tag:
+        if tag.name is None:
+            text += tag.text
+        tag = tag.next_element
+    return text
+
+def get_json_from_file(file):
+    otr_file_contents = file.read_text()
+    otr_json = json.loads(otr_file_contents)
+    return otr_json
+
+def parse_html(html):
+    soup = BeautifulSoup(html, 'html.parser')
+    return soup
+
+def get_all_timestamps(soup):
+    return soup.findAll('span', {'class': 'timestamp'})
```

Comment:
i would keep those in a module called utils and import from that as it can be reuse in format otr module.

Reply:
working on that

Reply:
done